src/cli.ts

```diff
@@ -38,6 +38,16 @@ function commonOptions(args: Argv<{}>): Argv<CommonOptions> {
       type: 'boolean',
       describe: 'force fetching from server, bypass cache',
     })
+    .option('sort', {
+      alias: 's',
+      type: 'boolean',
+      describe: 'sort by most outdated',
+    })
+    .option('sortReversed', {
```

Comment:
Instead of two options, I prefer to have one `sort` option like:

```
--sort time-asc
--sort time-desc
--sort diff-asc
--sort diff-desc
```

Reply:
Check it again! :)